analyses/cell-type-wilms-tumor-06/run.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+
+# parse config parameters:
+source bash/parse_yaml.sh
+eval $(parse_yaml config.yaml CONF_)
```

Comment:
I'm not sure if we expect everyone who might want to run this module to have access to `bash/parse_yaml.sh`?

Reply:
... Start being a bit too high-level for me to be honnest... 
All our template have been done by our expert Christoph Hafemeister, and here, I have to admit, I run it without understanding every line... 

I'll ask him once he is back from vacation!

Would this replace the run.sh and config.yaml files in the end?
https://openscpca.readthedocs.io/en/latest/ensuring-repro/docker/docker-images/#r-based-images
![image](https://github.com/user-attachments/assets/b4125376-d2da-457b-af9a-fc5cc29fbf63)